backend.config.js

```diff
@@ -1,21 +1,34 @@
+// Also wss://kusama-rpc.polkadot.io
+const DEFAULT_WS_PROVIDER_URL = "ws://kusama-rpc.polkadot.io";
```

Comment:
Yeeeey!!! :tada: Finally

Reply:
Sorry, actually it was a mistake. I didn't intend to make this change.

Reply:
Small thing. There is an `s` missing in `wss` here